Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -70,7 +70,7 @@ internal extension Valet {
 
     // MARK: Shared Access Group
 
-    static var sharedAccessGroupIdentifier: SharedGroupIdentifier = {
+    static let sharedAccessGroupIdentifier: SharedGroupIdentifier = {
```

Comment:
these were never set. not sure why I made them `var` to begin with, but... easy fix!